lib/smack/SmackRep.cpp

```diff
@@ -1012,13 +1017,44 @@ std::string SmackRep::code(llvm::CallInst& ci) {
 
   std::string s = fmt;
   for (unsigned i=1; i<ci.getNumOperands()-1; i++) {
-    const Expr* a = arg(f, i, ci.getOperand(i));
+    Value* argV = ci.getOperand(i);
     std::string::size_type idx = s.find('@');
     assert(idx != std::string::npos && "inline code: too many arguments.");
 
+    llvm::Type* targetType = argV->getType();
+    bool isCast = false;
+    if (idx + 1 < s.length()) {
+      switch(s[idx+1]) {
```

Comment:
Wrong formatting again for switch-case - indentation is wrong. See here for how to format it according to LLVM coding style that we also try to follow: https://github.com/llvm-mirror/llvm/blob/master/lib/IR/ConstantRange.cpp#L83